docs/walkthroughs/connecting-device-to-service.adoc

```diff
@@ -0,0 +1,211 @@
+[[mobile-core-public-ip]]
+= Mobile Core installation with public IP
+To start openshift cluster for local development and connect your device to same network you need to specify public IP of
+your laptop or server you want openshift being installed and provide following parameters
+```
+Use public IP yes/no?: _yes_
+Cluster IP local/public: <your-eth0-ip>|<your-wlan0-ip>
+```
+image::images/mobile-core-public-ip-setup.png[openshift-mobile-core-public-ip-setup]
+Find IP address of your local machine you can with this command ```ifconfig | grep inet```.
+Output should look like this
+```
+	inet 127.0.0.1 netmask 0xff000000
+	inet 10.40.3.70 netmask 0xfffffe00 broadcast 10.40.3.255
+```
+When installation is finished and successful you should get same result as on image below
+
+image::images/mobile-core-public-ip-success.png[openshift-mobile-core-public-ip-success]
+Web console URL should contain your public IP address in this case it is `10.40.3.70`
+
+[[mobile-core-services]]
+= Provision Mobile services in Openshift
+Now we need to login into _https://10.40.3.70:8443/console_, create project e.g. _test1_ and provision mobile services
+Let us provision Android mobile client from https://github.com/aerogear/android-showcase-template[android-showcase-template] application in _test1_ project
+
+image::images/mobile-client-provision-android.png[openshift-mobile-client-provision-android]
+Also we need to provision KeyCloak, Unified Push and Metrics mobile services in _test1_ project
+
+image::images/mobile-core-services-provisioned.png[openshift-mobile-core-services-provisioned]
+After all services are successfuly provisioned, next step is to bind them with our mobile client
+Open _android-showcase-template_ application under _Mobile Clients_ view and go to Mobile Services tab
+
+image::images/mobile-client-services-bindings.png[openshift-mobile-client-services-bindings]
+Create bindings to KeyCloak, Metrics and Unified Push services. For Unified Push server you need provide 2 parameters,
+_Server key_[Your Google Key for Firebase Cloud Messaging] and _Sender Id_[Your FCM Project Number, needed on the mobile client for connecting with FCM]
+You can obtain these parameters from https://console.firebase.google.com[Firebase] console after creating new project
+
+image::images/mobile-client-firebase-parameters.png[openshift-mobile-client-firebase-parameters]
```

Comment:
Do you want to show the full server key in this screenshot?

Reply:
I am thinking about that if I should hide some information